go/vt/sqlparser/comments.go

```diff
@@ -49,6 +49,8 @@ const (
 	DirectiveAllowHashJoin = "ALLOW_HASH_JOIN"
 	// DirectiveQueryPlanner lets the user specify per query which planner should be used
 	DirectiveQueryPlanner = "PLANNER"
+	// DirectiveAllowBlockJoin allows the planner to use BLOCK JOINS when possible.
+	DirectiveAllowBlockJoin = "ALLOW_BLOCK_JOIN"
```

Comment:
should we do BLOCK_JOIN=`<bool_value>` to make is possible to turn off if default is `on` or does it already work like that?

Reply:
I rather keep it as simple as possible. Do you think this is needed?